orienteer-core/src/main/java/org/orienteer/core/component/widget/document/ODocumentPropertiesWidget.java

```diff
@@ -22,14 +23,17 @@
 import org.orienteer.core.widget.AbstractModeAwareWidget;
 import org.orienteer.core.widget.Widget;
 
-import com.google.inject.Inject;
-import com.orientechnologies.orient.core.metadata.schema.OProperty;
-import com.orientechnologies.orient.core.record.impl.ODocument;
+import java.util.List;
 
 /**
  * Widget to show registered parameters for a document on particular tab
  */
-@Widget(domain="document", id = ODocumentPropertiesWidget.WIDGET_TYPE_ID, autoEnable=true)
+@Widget(
+		domain="document",
+		id = ODocumentPropertiesWidget.WIDGET_TYPE_ID,
+		tab = ODocumentPropertiesWidget.WIDGET_TYPE_ID,
```

Comment:
As discussed, the widget should be available on any tab, so please do not attach the widget to a particular one.

Reply:
Understood. Thanks